### ISSUE TITLE
Type 'HIChartView' has no member 'addFont'

### ISSUE THREAD

**Issue by TillitForsikringAS** [highcharts/highcharts-ios#346]
Hi, I'm trying to add a custom font to my Highchart graph, but it doesn't seem like HIChartView.addFont(fontPath) exist.

I've tested this code (following https://github.com/highcharts/highcharts-ios/blob/master/README.md#custom-fonts):
```
let fontPath = Bundle.main.path(forResource: "Windsong", ofType: "ttf")
HIChartView.addFont(fontPath)
```

I'm using the following pod: 
`pod 'Highcharts', '~> 6.1.0'`

**Comment by ihnatmoisieiev** [comment]
Hello @TillitForsikringAS,
this feature is available from [`7.0.2`](https://github.com/highcharts/highcharts-ios/blob/12e9eaf4409a7c452743bb660cec78dfecda59ad/CHANGELOG#L7) version.

**Comment by TillitForsikringAS** [comment]
Thanks!
